specification/help/resource-manager/readme.go.md

```diff
@@ -0,0 +1,11 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go) && $(track2)
+azure-arm: true
+license-header: MICROSOFT_MIT_NO_VERSION
+module-name: sdk/resourcemanager/help/armhelp
```

Comment:
Have you got the module name and package name reviewed?

Reply:
Not yet. Where should I get it reviewed?

Reply:
Please follow this:

Follow this guidance [Naming for new initial management or client libraries (new SDKs) - Overview (azure.com)](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/821/Naming-for-new-initial-management-or-client-libraries-(new-SDKs)) to create an issue for management client library name arch board review.

Reply:
Can you also share overall guidance for sdk generation. Is SDK generation tied to this swagger pr or can they be created at a later date. We are targeting launching documentation site end of this month (March)

Reply:
@lirenhe - Opened the naming review https://github.com/Azure/azure-sdk/issues/5817. Can we merge this pr so that we can generate the documentation website and I'll work on Sdk tasks parallely.